### ISSUE TITLE
'Back' and 'Cancel'

### ISSUE THREAD

**Issue by souperneon** [transitland/feed-registry#176]
- Figure out the logic for back and cancel to exist and for which steps they show up
- Maybe add a confirmation modal when user clicks any of these in the middle of the flow where they might lose information

**Comment by drewda** [comment]
@souperneon you all addressed this on other tickets in the meantime, right?

**Comment by souperneon** [comment]
Nah. This is for a future review. Right now there is no cancel  and we talked about adding that which would require additional modals/warning messages.
